packages/cli/src/evaluation.ee/test-runner/test-runner.service.ee.ts

```diff
@@ -63,6 +65,68 @@ export class TestRunnerService {
 		return workflow.nodes.find((node) => node.type === EVALUATION_DATASET_TRIGGER_NODE);
 	}
 
+	/**
+	 * Validates the evaluation trigger node is present in the workflow
+	 * and is configured correctly.
+	 */
+	private validateTriggerNode(workflow: IWorkflowBase) {
+		const triggerNode = this.findTriggerNode(workflow);
```

Comment:
nitpick - this could also be clearer since it finds only one type of node `findEvaluationTriggerNode`